codespell_lib/data/dictionary.txt

```diff
@@ -10967,9 +10967,9 @@ overreide->override
 overreides->overrides
 overriabled->overridable
 overriddden->overridden
-overridded->overridden
+overridded->overrode. overridden,
```

Comment:
comma not period

Reply:
Oh, I missed that before I hit approved. No way to undo my approval.